config/setupTests.js

```diff
@@ -8,3 +8,13 @@ global.shallow = shallow;
 global.render = render;
 global.mount = mount;
 global.React = React;
+global.insights = {
+    chrome: {
+        visibilityFunctions: {
+            something: (...args) => Boolean(args && args.length > 0 && args[0])
```

Comment:
this seems a bit strange `args.length > 0 && args[0]`. I think either of those conditions is enough

Reply:
This is for testing purposes so I can pass
```
permissions: {
  method: 'something'
  args: [false]
}
```

and test that it's actually false

Reply:
Ok